css/index.css

```diff
@@ -137,10 +191,156 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+@media (max-width: 1000px) {
+    .middle-content .boxes .box2 {
+        background: forestgreen;
+    }
+
+    .middle-content .boxes .box4 {
+        background: darkorchid;
+    }
+
+    .middle-content .boxes .box6 {
+        background: hotpink;
+    }
+
+    .middle-content .boxes .box8 {
+        background: indigo;
+    }
+
+    .middle-content .boxes .box10 {
+        background: lawngreen;
+    }
+}
+
+
+@media (max-width: 768px) {
+
+    header {
+        flex-direction: column;
+        flex-wrap: wrap;
+        align-items: center;
+        margin-bottom: 4.5%;
+    }
+
+    header .logo {
+        align-self: center;
+    }
+
+    .top-content {
+        margin-top: 2.7%;
+    }
+
+    .middle-content .boxes .box {
+        width: 13%;
+    }
+
+    .middle-content .boxes .box3 {
+        background: gold;
+    }
+
+    .middle-content .boxes .box5 {
+        background: cadetblue;
+    }
+
+    .middle-content .boxes .box7 {
+        background: coral;
+    }
+
+    .middle-content .boxes .box9 {
+        background: crimson;
+    }
+
+}
+
+
+
+@media (max-width: 400px) {
+
+    header {
+        width: 100%
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+
+    header nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    header nav ul {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        margin-top: 5px;
+        margin-bottom: 5px;
+        align-items: center;
+    }
+
+    header img {
+        width: 48%;
+    }
+
+    header nav ul li {
+        width: 100%;
+        height: 45px;
+        padding: 4%;
+        text-align: center;
+        border-top: 1px solid black;
+    }
+
+    .top-content {
+        flex-direction: column;
+        padding-bottom: 3%;
+    }
+
+    .top-content .jumbo {
+        width: 100%;
+        margin: 0;
+    }
+
+    .top-content .text-container {
+        font-size: 1.8rem;
+        width: 100%;
+        padding: 3% 6% 2%;
+    }
+
+    .middle-content .boxes {
+        flex-direction: row-reverse;
+    }
+
+    .middle-content .boxes .box {
```

Comment:
You can give the boxes a height of `100px` so that they keep their square shape!